pennylane/labs/tests/vibrational_ham/test_bosonic_mapping.py

```diff
@@ -0,0 +1,255 @@
+                X(0) @ X(2) @ Y(3),
+                X(0) @ Y(2) @ X(3),
+                X(0) @ Y(2) @ Y(3),
+                X(0) @ Z(1) @ X(2),
+                X(0) @ Z(1) @ X(2) @ Z(3),
+                X(0) @ Z(1) @ Y(2),
+                X(0) @ Z(1) @ Y(2) @ Z(3),
+                X(0) @ Z(1) @ X(2) @ X(3),
+                X(0) @ Z(1) @ X(2) @ Y(3),
+                X(0) @ Z(1) @ Y(2) @ X(3),
+                X(0) @ Z(1) @ Y(2) @ Y(3),
+                Y(0) @ X(2),
+                Y(0) @ X(2) @ Z(3),
+                Y(0) @ Y(2),
+                Y(0) @ Y(2) @ Z(3),
+                Y(0) @ X(2) @ X(3),
+                Y(0) @ X(2) @ Y(3),
+                Y(0) @ Y(2) @ X(3),
+                Y(0) @ Y(2) @ Y(3),
+                Y(0) @ Z(1) @ X(2),
+                Y(0) @ Z(1) @ X(2) @ Z(3),
+                Y(0) @ Z(1) @ Y(2),
+                Y(0) @ Z(1) @ Y(2) @ Z(3),
+                Y(0) @ Z(1) @ X(2) @ X(3),
+                Y(0) @ Z(1) @ X(2) @ Y(3),
+                Y(0) @ Z(1) @ Y(2) @ X(3),
+                Y(0) @ Z(1) @ Y(2) @ Y(3),
+                X(0) @ X(1) @ X(2),
+                X(0) @ X(1) @ X(2) @ Z(3),
+                X(0) @ X(1) @ Y(2),
+                X(0) @ X(1) @ Y(2) @ Z(3),
+                X(0) @ X(1) @ X(2) @ X(3),
+                X(0) @ X(1) @ X(2) @ Y(3),
+                X(0) @ X(1) @ Y(2) @ X(3),
+                X(0) @ X(1) @ Y(2) @ Y(3),
+                X(0) @ Y(1) @ X(2),
+                X(0) @ Y(1) @ X(2) @ Z(3),
+                X(0) @ Y(1) @ Y(2),
+                X(0) @ Y(1) @ Y(2) @ Z(3),
+                X(0) @ Y(1) @ X(2) @ X(3),
+                X(0) @ Y(1) @ X(2) @ Y(3),
+                X(0) @ Y(1) @ Y(2) @ X(3),
+                X(0) @ Y(1) @ Y(2) @ Y(3),
+                Y(0) @ X(1) @ X(2),
+                Y(0) @ X(1) @ X(2) @ Z(3),
+                Y(0) @ X(1) @ Y(2),
+                Y(0) @ X(1) @ Y(2) @ Z(3),
+                Y(0) @ X(1) @ X(2) @ X(3),
+                Y(0) @ X(1) @ X(2) @ Y(3),
+                Y(0) @ X(1) @ Y(2) @ X(3),
+                Y(0) @ X(1) @ Y(2) @ Y(3),
+                Y(0) @ Y(1) @ X(2),
+                Y(0) @ Y(1) @ X(2) @ Z(3),
+                Y(0) @ Y(1) @ Y(2),
+                Y(0) @ Y(1) @ Y(2) @ Z(3),
+                Y(0) @ Y(1) @ X(2) @ X(3),
+                Y(0) @ Y(1) @ X(2) @ Y(3),
+                Y(0) @ Y(1) @ Y(2) @ X(3),
+                Y(0) @ Y(1) @ Y(2) @ Y(3),
+            ],
+        ),
+    ),
+]
+
+
+@pytest.mark.parametrize("bose_op, d, result", BOSE_WORDS_AND_OPS)
+def test_binary_mapping_boseword(bose_op, d, result):
+    """Test that the binary_mapping function returns the correct qubit operator."""
+    # convert BoseWord to PauliSentence and simplify
+    qubit_op = binary_mapping(bose_op, d=d)
+    qubit_op.simplify(tol=1e-8)
+
+    # get expected op as PauliSentence and simplify
+    expected_op = pauli_sentence(qml.Hamiltonian(result[0], result[1]))
+    expected_op.simplify(tol=1e-8)
```

Comment:
You can do this simplification in the test case itself and rename result to expected op I think. A little bit of extra work on your end though 😅